### PR TITLE
cfp/8: Add CybelAngel GKE NAT talk

### PR DESCRIPTION
**Pull request by Azraeht**
Hi!

Here is a proposal for a  15/20 talk for the next SRE Meetup

### CODE REVIEW

File: cfp/#7/gke-nat.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+__title__: Building Custom NAT for GKE
+
+__expected time__ : 15/20m
+
+__abstract__: GCP offers a powerfull but expansive Cloud NAT product which is fully managed. After several months using it, it appears its cost was too high for this feature simple feature. We dedide to build our own NAT gateway on top of our GKE cluster.
```
Comment (tormath1) [inline cfp/#7/gke-nat.md:5]:
```suggestion
__abstract__: GCP offers a powerful but expansive Cloud NAT product which is fully managed. After several months using it, it appears its cost was too high for this feature simple feature. We decided to build our own NAT gateway on top of our GKE cluster.
```

**Review by tormath1 — COMMENTED** [review]
Super cool ! I'm looking forward to see this talk. :) I just added some grammar fix suggestion :wink:

File: cfp/#7/gke-nat.md (hunk 1 above)
Comment (Azraeht) [reply cfp/#7/gke-nat.md]: Integrated :)

**Review by pabluk — APPROVED** [review]
LGTM! :+1:

**Pull request merged**